cmd/podman/varlink/io.projectatomic.podman.varlink

```diff
@@ -391,8 +391,14 @@ method SearchImage(name: string, limit: int) -> (images: []ImageSearch)
 # in a string array.
 method DeleteUnusedImages() -> (images: []string)
 
-# This method is not implemented.
-method CreateFromContainer() -> (notimplemented: NotImplemented)
+# CreateImageFromContainer, also known as Commit, creates an image from an existing container. It requires the name or
+# ID of the container as well as the resulting image name.  Optionally, you can define an author and message
+# to be added to the resulting image.  You can also define changes to the resulting image for the following
+# attributes: _CMD, ENTRYPOINT, ENV, EXPOSE, LABEL, STOPSIGNAL, USER, VOLUME, and WORKDIR_.  To pause the
+# container while it is being committed, pass a _true_ bool for the pause argument.  If the container cannot
+# be found by the ID or name provided, a (ContainerNotFound)[#ContainerNotFound] error will be returned; otherwise,
+# the resulting image's ID will be returned as a string.
+method CreateFromContainer(name: string, image_name: string, changes: []string, author: string, message: string, pause: bool) -> (image: string)
```

Comment:
CreateFromContainer or CreateImageFromContainer?

Reply:
it is commit